tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["**/*.ts", "**/*.js"]
+}
```

Comment:
I'm not sure this is the perfect solution but it means that eslint can lint any file in our repo, Typescript or Javascript. As aforementioned this is specifically here for `vitest.config.js`.